### ISSUE TITLE
Time since latest version is incorrect if the gem has pre-releases

### ISSUE THREAD

**Issue by AaronC81** [zverok/whatthegem#4]
First off, this is a really cool project! Thanks for creating it, I can see this coming in really handy :smile:

I tried it out on one of my gems (Parlour), and spotted a tiny bug. On the `Latest version: ` line, it seems to show the name of the latest _stable_ version, but the time since the last _pre-release_ version. So, for Parlour, it shows:

```
Latest version: 4.0.1 (1 week ago)
```

4.0.1 actually came out in August, but there was a pre-release version (5.0.0.beta.6) which was in fact published to RubyGems a week ago.

I think there are two ways to fix this: it should either should show the pre-release version number here, or show the correct time since 4.0.1 was released. If you decide which way you'd prefer, I'm happy to put together a PR for this if you like :)

**Comment by zverok** [comment]
Thanks for reporting! I haven't considered prereleases, indead. Fixed in `master`, please check if this makes sense to you:
```
$ ./exe/whatthegem parlour
parlour
┃  An RBI generator, merger and parser for Sorbet
┃  (https://github.com/AaronC81/parlour)

  Info

          Latest version: 4.0.1 (2 month ago)
              Prerelease: 5.0.0.beta.6 (1 week ago)
```
(The last line would be absent if the freshest version is not a prerelease)

**Comment by AaronC81** [comment]
Yeah, that looks really clear to me! Thanks :)

**Comment by zverok** [comment]
Released as 0.0.4, sorry for the delay.

**Comment by AaronC81** [comment]
No worries at all! Thank you very much :)
